contracts/src/TokenHome/ERC20TokenHome.sol

```diff
@@ -27,8 +27,25 @@ import {CallUtils} from "../utils/CallUtils.sol";
 contract ERC20TokenHome is IERC20TokenHome, TokenHome {
     using SafeERC20 for IERC20;
 
-    /// @notice The ERC20 token this home contract transfers to TokenRemote instances.
-    IERC20 public immutable token;
+    // solhint-disable private-vars-leading-underscore
+    /// @custom:storage-location erc7201:avalanche-ictt.storage.ERC20TokenHome
```

Comment:
I think these structs should have a `@dev` to explain why we're using namespaced storage.

Reply:
updated